utils/src/ast-grep/package-json.test.ts

```diff
@@ -0,0 +1,175 @@
+import assert from "node:assert/strict";
+import { describe, it } from "node:test";
+import dedent from "dedent";
+import jsonLang from "@ast-grep/lang-json";
+import { registerDynamicLanguage, parse } from "@ast-grep/napi";
+import type { Edit } from "@ast-grep/napi";
+import { getScriptsNode, getNodeJsUsage, replaceNodeJsArgs } from "./package-json.ts";
+
+registerDynamicLanguage({ json: jsonLang });
+
+describe("package-json utilities", () => {
+	describe("getScriptsNode", () => {
+		it("should get the scripts node", () => {
+			const input = dedent`
+				{
+					"scripts": {
+						"test": "echo \\"Error: no test specified\\" && exit 1"
```

Comment:
These double backslashes are escaping the backslash itself instead of the quotes, making this JSON invalid. I think the correct form is with just one.
```suggestion
						"test": "echo \"Error: no test specified\" && exit 1"
```